packages/lesswrong/components/thinkPage/ThinkOmnibar.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState, useEffect } from 'react';
+import { Components, registerComponent } from '../../lib/vulcan-lib';
+import { useTracking } from "../../lib/analyticsEvents";
+import Input from '@material-ui/core/Input';
+import { useLazyQuery } from '@apollo/client';
+import gql from 'graphql-tag';
```

Comment:
I think this should be imported from `@apollo/client` as well?  Not sure if anything goes wrong, but that's how we do it in most places